### ISSUE TITLE
inRandomOrder() Function + Take Function

### ISSUE THREAD

**Issue by ablishek** [spatie/laravel-query-builder#446]
Hi,

Is there anyway we can extend and access the **inRandomOrder()**
Similar to what can be accomplished by Count, but rather allow the end user to specify if they wish to receive data in random order or not.

Additionally I know pagination is allowed, but can we specify the quantity of records we wish to recieve simply by assigning a TAKE(10), or do you suggest we accomplish this by pagination?

**Comment by AlexVanderbist** [comment]
Hi! 

>Is there anyway we can extend and access the inRandomOrder()

You can create a custom sort that simply calls the `inRandomOrder()` method on the query. The resulting request could look like this: `GET /users?sort=random`. Read more here: https://docs.spatie.be/laravel-query-builder/v2/features/sorting/#custom-sorts

>Additionally I know pagination is allowed, but can we specify the quantity of records we wish to receive simply by assigning a TAKE(10), or do you suggest we accomplish this by pagination?

The query builder only builds queries (what a surprise), what you want to do with the query is up to you. You can `paginate()`, `get()`, `first()` or if you need just 10 results, `take(10)` would work just fine :) 

Feel free to open a new issue if you need help with that custom sort class 👍

**Comment by ablishek** [comment]
Sorry I wasn't clear enough. When I said pagination, I actually mean in realation to the above random function for included relationships. 

Meaning, that if I got related posts, I could limit those posts by 10 instead of bringing all of them onto view. 

For now I have been able to accomplish this by assigning a special relationship in the model called randomPosts where I run random plus take function.
